package.json

```diff
@@ -56,8 +57,5 @@
     "*.scss": [
       "stylelint --fix"
     ]
-  },
-  "nx": {
-    "includedScripts": []
```

Comment:
It was defining `time-out-market` itself as a module, but it is a workspace/ecosystem